Lucene.Net.DocumentMapper.Tests/Lucene.Net.DocumentMapper.Tests.csproj

```diff
@@ -1,28 +1,41 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<Nullable>enable</Nullable>
+		<LangVersion>preview</LangVersion>
+		<IsPackable>false</IsPackable>
+	</PropertyGroup>
 
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
+	<ItemGroup>
```

Comment:
This item group is oddly specific to your machine, can you remove it?

Reply:
@huster-songtao mind removing this item group?

Reply:
Yeah I can't remember why I put that on the test project, I think I may have had issues where it was creating a nuget package for the test project as well.